.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,10 @@
+<!-- ************************** IMPORTANT ************************* -->
+<!-- ** Continuous Deployment is enabled for this repository     ** -->
+<!-- ** Merging into master = deploying to PROD                  ** -->
+<!-- **                                                          ** -->
+<!-- ** Use Riff-Raff to deploy/test a PR on CODE before merging ** -->
```

Comment:
I wanted this message to be quite short and to the point. I will reiterate your point on Slack/email tomorrow when we enable CD.